File: OLSource1/v1/VS_officedev/codesnippet/CSharp/walkthrough--updating-the-controls-on-a-ribbon-at-run-time_8.cs
        private void PopulateMailItem(string addressToLine)
        {
            Outlook.Application application = Globals.ThisAddIn.Application;
            Outlook.Inspector inspector = application.ActiveInspector();
            Outlook.MailItem myMailItem = (Outlook.MailItem)inspector.CurrentItem;

            myMailItem.To = "";
            String[] tempArray = addressToLine.Split(new Char[] { '|' });
            myMailItem.To = tempArray[0] + "@example.com";
            myMailItem.Subject = "Following up on your order";
            myMailItem.Body = "Hello " + tempArray[0] + "," +
                "\n" + "We would like to get your feedback on the " +
                "following products that you recently ordered: ";
        }

File: OLSource1/v1/VS_officedev/codesnippet/VisualBasic/walkthrough--updating-the-controls-on-a-ribbon-at-run-time_8.vb
    Private Sub PopulateMailItem(ByVal addressToLine As String)
        Dim application As Outlook.Application = Globals.ThisAddIn.Application
        Dim inspector As Outlook.Inspector = application.ActiveInspector()
        Dim myMailItem As Outlook.MailItem = _
            CType(inspector.CurrentItem, Outlook.MailItem)

        myMailItem.To = ""
        Dim tempArray As [String]() = addressToLine.Split(New [Char]() {"|"c})
        myMailItem.To = tempArray(1) + "@example.com"
        myMailItem.Subject = "Following up on your order"
        myMailItem.Body = "Hello " + tempArray(1) + "," _
            + ControlChars.Lf + "We would like to get your feedback" + _
            "on the following products that you recently ordered: "
    End Sub